net/unbound/files/root.hints

```diff
@@ -0,0 +1,39 @@
+.			518400	IN	NS	a.root-servers.net.
```

Comment:
This file should be left for unbound to update. It includes a copy of the roots and will regularily update them ([unbound-anchor](https://www.unbound.net/documentation/unbound-anchor.html)). This is the reason I removed it a few versions back.